### PR TITLE
Druid InsectSwarm also fixes a bug with moonfire min/max damage

### PR DESCRIPTION
**Pull request by ElliotWood**

### CODE REVIEW

File: sim/druid/moonfire.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func (druid *Druid) getMoonfireBaseConfig(rank int) core.SpellConfig {
 	spellCoeff := MoonfiresSpellCoeff[rank]
 	spellDotCoeff := MoonfiresSellDotCoeff[rank]
 	baseDamageLow := MoonfireBaseDamage[rank][0]
-	baseDamageHigh := MoonfireBaseDamage[rank][0]
+	baseDamageHigh := MoonfireBaseDamage[rank][1]
```
Comment (kayla-glick) [inline sim/druid/moonfire.go:37]: Oooooh dang it I must've missed that!

File: sim/druid/insect_swarm.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	druid.InsectSwarm = make([]*DruidSpell, InsectSwarmRanks+1)
+
+	for rank := 1; rank <= InsectSwarmRanks; rank++ {
+		config := druid.getInsectSwarmBaseConfig(rank)
+
+		if config.RequiredLevel <= int(druid.Level) {
+			druid.InsectSwarm[rank] = druid.RegisterSpell(Humanoid|Moonkin, config)
+		}
+	}
+}
```
Comment (kayla-glick) [inline sim/druid/insect_swarm.go:28]: 🔨 We should probably only register them if the druid has the insect swarm talent selected

File: sim/druid/insect_swarm.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+				dot.SnapshotAttackerMultiplier = 1 // dot.Spell.AttackerDamageMultiplier(dot.Spell.Unit.AttackTables[target.UnitIndex])
+			},
+			OnTick: func(sim *core.Simulation, target *core.Unit, dot *core.Dot) {
+				dot.CalcAndDealPeriodicSnapshotDamage(sim, target, dot.OutcomeTick)
+			},
+		},
+
+		BonusCritRating:  core.SpellCritRatingPerCritChance,
+		DamageMultiplier: 1,
+		CritMultiplier:   1,
```
Comment (kayla-glick) [inline sim/druid/insect_swarm.go:74]: Are the crit modifiers needed since Insect Swarm can't crit at the moment?

**Comment by kayla-glick** [comment]
Hey @ElliotWood if you're around would you mind trying to wrap this one up? I'd like to try to get this merged in for the druid sim

File: sim/druid/insect_swarm.go (hunk 2 above)
Comment (ElliotWood) [reply sim/druid/insect_swarm.go:28]: Sorry I was MIA - shall I close this one?

**Comment by ElliotWood** [comment]
Got too busy to finish. Apologies, might be able to pick it up in a couple of weeks

**Pull request closed without merging**